Godeps/Godeps.json

```diff
@@ -504,18 +504,6 @@
 			"Rev": "f368244301305f414206f889b1735a54cfc8bde8"
 		},
 		{
-			"ImportPath": "github.com/spf13/cobra/cobra",
```

Comment:
Hm that's weird -- I'm fine with deleting those deps in this PR or a separate one.  It seems like the actual source files for these deps are still in our `vendor` directory though.